src/pocketmine/entity/FloatingText.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace pocketmine\entity;
+
+use pocketmine\item\Item as ItemItem;
+use pocketmine\network\protocol\AddPlayerPacket;
+use pocketmine\Player;
+use pocketmine\utils\UUID;
+
+class FloatingText extends Entity {
+    const NETWORK_ID = 80;
```

Comment:
I know. You need to set a network ID for an entity, otherwise, the server won't register it in the known entities list. As I don't use the ID in the code, it's completely meaningless, just a number higher than -1 needed for the server.